src/frontend/src/lib/components/auth/LandingPage.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import PreviewLarge from '$lib/assets/preview-large.webp';
```

Comment:
for consistency I would suggest to name the imported image with lowercase. If I'm not wrong, didnt double checked the code, it's what we generally do when we import `$lib/assets...`

i.e. only component with capital letter.

Reply:
You are right, corrected!